README.md

```diff
@@ -98,3 +98,17 @@ Populate inventory list of IPs of managed nodes to provision:
 Command to provision a VM from an Ansible Control Node
 
     ec2-user@ControlNode:/home/ec2-user $ ansible-playbook .ansible/playbook2.yaml -i .ansible/inventory.ini
+
+
+## Docker
+
+You can build the two different environment containers with the appropriate command:
+```bash
+docker build --target production --tag todo-app:prod .
+docker build --target development --tag todo-app:dev .
+```
+And run with:
+```bash
+docker run -d --env-file .env.docker -p 8000:8000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:prod
+docker run -d --env-file .env -p 5000:5000 todo-app:dev
```

Comment:
Minor but I'd swap the bind mount from the first to the second line here - really we want it on our dev container since that is where we expect to be changing files outside the container and want the hot reloading behaviour:
```suggestion
docker run -d --env-file .env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
```